public/static/docs/command-reference/config.md

```diff
@@ -136,9 +127,10 @@ for more details.) This section contains the following options:
   to protect user from accidental cache and repository corruption.
 
   ⚠️ If you manually set `cache.type` to `hardlink` or `symlink`, **you will
-  corrupt the cache** if you modify tracked data files in the workspace. See the
-  `cache.protected` option above, and corresponding `dvc unprotect` command to
-  modify files safely.
+  corrupt the cache** if you modify tracked data files in the workspace. In
+  an attempt to prevent that, DVC will automatically make those links
+  read-only and an additional barrier. Use `dvc unprotect` command to modify
+  files safely.
```

Comment:
```suggestion
  an attempt to prevent that, DVC will automatically protect those file links
  (make them read-only). Use `dvc unprotect` to be able to modify them safely.
```

> safely

Does `dvc unprotect` ensure this operation is safe though?

Reply:
@jorgeorpinel yes.